service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/RepresentativesInbox.java

```diff
@@ -91,39 +85,7 @@ public HashSet<String> getChildrenSolicitorEmails(CaseData caseData,
             .collect(Collectors.toCollection(LinkedHashSet::new));
     }
 
-    public Set<Recipient> getNonSelectedRespondentRecipientsByPost(CaseData caseData,
-                                                                   List<Element<Respondent>> respondentsSelected) {
-        Set<Recipient> nonSelectedRespondentsRecipients = getNonSelectedRespondentRecipients(
-            POST, caseData, respondentsSelected, Element::getValue
-        );
-
-        nonSelectedRespondentsRecipients.addAll(getNotSelectedUnrepresentedRespondents(caseData, respondentsSelected));
-
-        return nonSelectedRespondentsRecipients;
-    }
-
-    public <R> Set<R> getNonSelectedRespondentRecipients(RepresentativeServingPreferences servingPreferences,
-                                                         CaseData caseData,
-                                                         List<Element<Respondent>> respondentsSelected,
-                                                         Function<Element<Representative>, R> mapperFunction) {
-        Set<UUID> allRepresentativeIds = unwrapElements(caseData.getAllRespondents())
-            .stream()
-            .flatMap(respondent -> respondent.getRepresentedBy().stream().map(Element::getValue))
-            .collect(Collectors.toSet());
-
-        Set<UUID> selectedRepresentativeIds = unwrapElements(respondentsSelected).stream()
-            .flatMap(respondent -> respondent.getRepresentedBy().stream().map(Element::getValue))
-            .collect(Collectors.toSet());
-
-        return caseData.getRepresentativesElementsByServedPreference(servingPreferences)
-            .stream()
-            .filter(representativeElement -> allRepresentativeIds.contains(representativeElement.getId()))
-            .filter(not(representativeElement -> selectedRepresentativeIds.contains(representativeElement.getId())))
-            .map(mapperFunction)
-            .collect(Collectors.toCollection(LinkedHashSet::new));
-    }
-
-    public Set<Recipient> getSelectedRecipientsWithNoRepresentation(List<Element<Respondent>> selectedRespondents) {
+    public Set<Recipient> getRecipientsWithNoRepresentation(List<Element<Respondent>> selectedRespondents) {
```

Comment:
may it's just me ... but not sure this function is actually used anywhere? (can see references for getSelectedRecipientsWithNoRepresentation though)